### ISSUE TITLE
[Design] The color of the + icon in the locales popover should be Neutral500

### ISSUE THREAD

**Issue by yanniskadiri** [strapi/design-system#1811]
Current design

![Image](https://github.com/user-attachments/assets/52ce14ae-74a9-49ee-a4d5-438b4fd40bd8)

Expected design

![Image](https://github.com/user-attachments/assets/54a04f12-bb00-4e2f-a669-f1fe71584f4a)

**Comment by lucasboilly** [comment]
@HichamELBSI yes, we should change it. The reason is that now that our icons are bolder (because of our new library) we need a lighter color to balance, right now we see them too much compared to the text.

Also the text color + the cursor should be enough to differentiate the disabled state but if it isn't we can make the icons lighter when disabled (ex: `neutral300`).

**Comment by Adzouz** [comment]
Hi there,
I made the change to fix it on the component itself directly in the design system. There's another PR (opened by @lucasboilly 🙏 https://github.com/strapi/strapi/pull/23264) that I will use to update all existing menus in the CMS to natively use the startIcon / endIcon and also use the danger variant in order to get rid of all custom style.
Let me know if you need more info, closing it but it won't be fixed until next version of Strapi (or the one after).
🚀
